cider/tests/CiderNongroupbyAggTest.cpp

```diff
@@ -365,8 +343,7 @@ TEST_F(CiderNongroupbyAggArrowFormatTest, SingleKeyScalarTypeTest) {
       .sum_double = 0.0,
       .max_double = -std::numeric_limits<double>::max(),
       .min_double = std::numeric_limits<double>::max(),
-      .avg_double = 0.0,
-      .null = 1};
+      .null = 0};
```

Comment:
what's the change `.null = 1` to `.null = 0` for?

Reply:
All the results have only one row. This ".null" will be compared with null bit of 1st column, which is COUNT. As we know COUNT can't be null.

      EXPECT_EQ((int64_t)!CiderBitUtils::isBitSetAt(
                    out_batch->getChildAt(0)->as<ScalarBatch<int64_t>>()->getNulls(), 0),
                expect_result.null);